### DISCUSSION TITLE
Size of calculator app

### DISCUSSION THREAD

**Discussion by hafizhasant** [elementary/triage#204] [Q&A]
Hi.Can you please increase the size of the calculator app in double..it is too small..or else atleast make it resizable.

**Comment by lenemter** [comment]
Hello. Thanks for reporting. This is a duplicate of https://github.com/elementary/calculator/issues/219.

**Reply by hafizhasant** [reply]
tht is fine ..but the overall size of the calculator s small..so i requested if it was possible to increase the size.thanks btw...
